tests/sentry/api/endpoints/test_team_alerts_triggered.py

```diff
@@ -0,0 +1,158 @@
+from sentry.incidents.models import (
+    AlertRuleThresholdType,
+    IncidentActivity,
+    IncidentActivityType,
+    IncidentStatus,
+    IncidentTrigger,
+    TriggerStatus,
+)
+from sentry.models import ActorTuple
+from sentry.testutils import APITestCase
+from sentry.testutils.helpers.datetime import before_now
+
+
+class TeamAlertsTriggeredTest(APITestCase):
+    def test_simple(self):
+        project1 = self.create_project(
+            teams=[self.team], slug="foo"
+        )  # This project will return counts for this team
+        user_owned_rule = self.create_alert_rule(
+            organization=self.organization,
+            projects=[project1],
+            name="user owned rule",
+            query="",
+            aggregate="count()",
+            time_window=1,
+            threshold_type=AlertRuleThresholdType.ABOVE,
+            resolve_threshold=10,
+            threshold_period=1,
+            owner=ActorTuple.from_actor_identifier(self.user.id),
+        )
+        user_owned_incident = self.create_incident(status=20, alert_rule=user_owned_rule)
+        for i in range(1, 31):
+            trigger = self.create_alert_rule_trigger(user_owned_rule, i, 200)
+            trigger2 = self.create_alert_rule_trigger(user_owned_rule, -i, 200)
+            IncidentTrigger.objects.create(
+                incident=user_owned_incident,
+                alert_rule_trigger=trigger,
+                status=TriggerStatus.ACTIVE.value,
+                date_added=before_now(days=i),
+            )
+            IncidentTrigger.objects.create(
+                incident=user_owned_incident,
+                alert_rule_trigger=trigger2,
+                status=TriggerStatus.ACTIVE.value,
+                date_added=before_now(days=i),
+            )
```

Comment:
This is a lot of data to create. Do we actually need these trigger rows?

If we want to write 31 `IncidentActivity` rows, let's use `.bulk_create`

Reply:
Removed all the triggers and changed to bulk_create to create 8 activities